recipe/meta.yaml

```diff
@@ -155,6 +155,9 @@ outputs:
         # having not been migrated yet causing conflicts with the "libgdal" output
         # See: https://github.com/conda-forge/gdal-feedstock/pull/415
         - hdf5
+        # Workaround to prevent conflicts seen in 
+        #  https://github.com/conda-forge/gdal-feedstock/pull/543
+        - openssl
```

Comment:
@conda-forge/gdal, I realize this isn't an ideal fix but it did work for me in a local test build.  I don't fully understand the solver issue but I think it may have to do with the intricacies of how pins and multiple outputs work (or don't work) together.